packages/dom/src/focusable.js

```diff
@@ -69,7 +69,7 @@ function isValidFocusableArea( element ) {
 /**
  * Returns all focusable elements within a given context.
  *
- * @param {Element} context Element in which to search.
+ * @param {ParentNode} context Element in which to search.
```

Comment:
`ParentNode` isn't a valid type as far as I can tell. It would be `Node` I believe?